mrtt-ui/src/components/SiteBackgroundForm.js

```diff
@@ -46,7 +52,7 @@ const ProjectDetailsForm = () => {
     lawStatus: yup.string(),
     managementArea: yup.string(),
     protectionStatus: multiselectWithOtherValidation,
-    areStakeholdersInvolved: yup.string(),
+    areStakeholdersInvolved: yup.string().nullable(),
```

Comment:
interesting. gtk

Reply:
was getting a weird error/complaint about it being set to null somewhere (to be honest, I didn't have enough time to really dig into where this was happening). Maybe somewhere the empty string is throwing off the api - or maybe Mui is doing it